xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogStyle.xml

```diff
@@ -286,11 +278,24 @@ div.edit-categories {
   font-weight: bold;
 }
 .createPost {
-  background: transparent url("$xwiki.getSkinFile('icons/silk/add.png')") no-repeat scroll left center;
+  /* background: transparent url("$xwiki.getSkinFile('icons/silk/add.png')") no-repeat scroll left center; */
   font-size: 80%; 
   font-weight: bold;
-  padding-left: 18px;
-}</code>
+  padding-left: 9px;
```

Comment:
I suppose the old padding is not needed anymore, since the icon is not present in the CSS class anymore but directly inside the content (see: https://github.com/xwiki/xwiki-platform/pull/544/files#diff-2a9df579b070606f0441218eb6306107L838).

However, testing with both icon themes is still a good practice to ensure we have not broken anything.